src/Implementation/TypeDetector.php

```diff
@@ -103,6 +106,10 @@ private static function determineScalarTypeOrListIfPossible(string $class, Refle
         return $schemaType->withExamples(
             ...array_map(
                 static function (ValueObject $valueObject) {
+                    if (in_array(DateTimeValue::class, class_implements($valueObject), true)) {
+                        return $valueObject->toString();
```

Comment:
this is not needed if you use the toValue correct.

Reply:
True 🎉